GrandCentralBoard/Extensions/NSDate+DateFormatter.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  NSDate.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 13.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+
+private let yearMonthDayFormatter: NSDateFormatter = {
+    let formatter = NSDateFormatter()
+    formatter.dateFormat = "yyyy-MM-dd"
+    formatter.locale = NSLocale(localeIdentifier: "en_US_POSIX")
+    return formatter
+}()
+
+extension NSDate {
+    func yearMonthDayString() -> String {
```

Comment:
I think that name should indicate that this should be used only for parsing dates from the network responses (as you override locale) - formatter used for formatting user facing strings shouldn't override locale.